src/components/application_manager/test/mock_message_helper.cc

```diff
@@ -347,6 +350,41 @@ const VehicleData& MessageHelper::vehicle_data() {
   return MockMessageHelper::message_helper_mock()->vehicle_data();
 }
 
+bool MessageHelper::SendStopAudioPathThru(ApplicationManager& app_mngr) {
```

Comment:
@AGaliuzov, because it mock.cc for MessageHelper and absolutely similar to original MessageHelper. My task is move test from PASA to Genivi and make it work, that is why I changed Mock. Make changes in original in MessageHelper it isn't my task. I'm sorry, that I can't do it.